### ISSUE TITLE
Meteor-specific require prevents static extraction

### ISSUE THREAD

**Issue by caktux** [ethereum/web3.js#289]
```
WARNING in ./~/web3/lib/web3/httpprovider.js
Critical dependencies:
28:84-91 require function is used in a way in which dependencies cannot be statically extracted
 @ ./~/web3/lib/web3/httpprovider.js 28:84-91
```

**Comment by caktux** [comment]
Changes in #284 fix this, please rebase, merge and publish a new version; been stuck on 0.8.1 since there hasn't been a single release that wasn't broken since then.

**Comment by debris** [comment]
fixed in 0.12.1

**Comment by caktux** [comment]
Indeed, thanks!
